VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+
+There are solutions that are considered to mitigate this issue. For example, we could create a sub-layer of delegation. Validator candidate could delegate to existing validators while they’re trying to attract delegators, thereby generating revenue. At the same time, delegators already bonded to validators could signal their intention to switch to this validator candidate should it get enough stake to enter the validator set. When the validator candidate would get enough pledge, it would transition to a validator and all the delegator who pledged would join its staking pool.
+
+### Do validators need to self-bond ATOMs?
+
+Delegators should want their validator to have self-bonded ATOMs in their staking pool; that is, bonded ATOMs of their own. In other words, validators should have skin-in-the-game. 
+
+In order for delegators to have some guarantee about how much skin-in-the-game their validator has, the latter must signal a minimum amount of self-bonded ATOMs. If a validator's self-bond goes below the limit that it predefined, this delegator and all of its delegators will unbond. 
+
+### How to prevent concentration of stake in the hands of a few top validators?
+
+For now we are expecting the community to behave in a smart and self-preserving way. When a mining pool in Bitcoin gets too much mining power the community usually stops contributing to that pool. We want to harness the same effect with Cosmos initially. We will put several mechanisms in place to smoothen this process as much as possible:
+
+* **Fast switching:** We want to allow delegators to easily switch from one validator to the other, in order to reduce validator stickiness 
+* **Hack bounty:** We want to incentivize the community to hack validators. There will be bounties proportionate to the size of the validator, so that a validator becomes bigger of a target as its stake grows.
+* **UI warning:** Users will be warned by the wallet’s UI if they want to delegate to a validator that already has a significant amount of staking power
+
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+It is highly probable that they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block rewards and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called `photons`.
+
+
+## Technical requirements 
+
+### What are hardware requirements ?
+
+Validators should expect to provision one or more data center locations with redundant power, networking, firewalls, HSMs and servers.
+We expect a modest level of hardware specifications will be needed initially and requirements to rise as network use rises. Participating in the testnet is the best way to learn more.
```

Comment:
Please maybe create new section which talks about making your hardware solar-flare proof, and EMP proof - check out this link to understand a bit more: http://www.preparednesspro.com/many-myths-emps-and-faraday-cages

Reply:
@zmanian

Reply:
@zmanian

Reply:
This doesn't need to addressed for this PR, but we should open up an issue to not forget about this